src/core/core.interaction.js

```diff
@@ -62,58 +62,68 @@ function getIntersectItems(chart, position) {
 }
 
 /**
- * Helper function to get the items nearest to the event position considering all visible items in teh chart
+ * Get a distance metric function for two points based on the
```

Comment:
it's strange to wrap the text after 60 cols. I'd put it all on one line

Reply:
I just moved this function and kept the comment as it was. But I'll change it in next commit.

Reply:
Fixed